### ISSUE TITLE
BUG: postgres dollar quote, year 2020

### ISSUE THREAD

**Issue by Andrei-Pozolotin** [dbeaver/dbeaver#7505]
#### System information: 
- Operating system (distribution) and version
```
Linux work3 5.4.6-arch3-1 #1 SMP PREEMPT Tue, 24 Dec 2019 04:36:53 +0000 x86_64 GNU/Linux
```
- DBeaver version
```
	JKISS	DBeaver IDE	6.3.1.201912221734	org.jkiss.dbeaver.core.eclipse
```
- Additional extensions
```
many
```

#### Connection specification:
- Database name and version
```
postgresql 11.5
```
- Driver name
```
postgresql-42.2.5.jar
```
- Do you use tunnels or proxies (SSH, SOCKS, etc)? 
```
no
```

#### Describe the problem you're observing:

1. snippet N1 executes fine both in `pgAdmin 4` and `dbeaver 6.3.1`
```sql
CREATE OR REPLACE FUNCTION system_quotes_verify(argument text)
RETURNS text LANGUAGE plpgsql AS $FUN$
BEGIN
    RETURN FORMAT($$
        hello-kitty
    $$);
END;
$FUN$;
```

2. snippet N2 executes fine in `pgAdmin 4` and fails in `dbeaver 6.3.1`
```sql
CREATE OR REPLACE FUNCTION system_quotes_verify(argument text)
RETURNS text LANGUAGE plpgsql AS $FUN$
BEGIN
    IF argument LIKE 'kitty_%' THEN
        RETURN FORMAT($$
            hello-kitty, %s;
        $$, argument);
    ELSE
        RETURN 'no-kitty';
    END IF;
END;
$FUN$;
```
with error:
```
SQL Error [42601]: Unterminated dollar quote started at position 96 in SQL CREATE OR REPLACE FUNCTION system_quotes_verify(argument text)
RETURNS text LANGUAGE plpgsql AS $FUN$
BEGIN
    IF argument LIKE 'kitty_%' THEN
        RETURN FORMAT($$
```

**Comment by serge-rider** [comment]
I believe it was fixed in #7460

**Comment by Andrei-Pozolotin** [comment]
yes, 
this is resolved in 
https://github.com/dbeaver/dbeaver/releases/tag/6.3.2 
thank you.
